api/types/filters/parse.go

```diff
@@ -199,6 +200,36 @@ func (args Args) Match(field, source string) bool {
 	return false
 }
 
+// ParseBoolIfPresent overwrites the target with a boolean value of the key if it's present.
+// If the value cannot be interpreted as a boolean an error is returned.
+func (args Args) ParseBoolIfPresent(target *bool, key string) error {
```

Comment:
Have you thought about returning a bool instead of dealing with the bool pointer?

Reply:
Yes - returning bool doesn't distinguish between value not being present and being false. In this case if value is missing, the target is not overwritten (leaves the previously assigned value) and error is nil.

I think we should go `...OrDefault` approach suggested by Sebastiaan though.